wallet_transaction.go

```diff
@@ -68,7 +68,7 @@ func (bmr *WalletTransactionRequest) Create(ctx context.Context, walletTransacti
 		return nil, err
 	}
 
-	walletTransactionResult := result.(*WalletTransaction)
+	walletTransactionResult := result.(*WalletTransactionResult)
```

Comment:
```suggestion
	walletTransactionResult, ok := result.(*WalletTransactionResult)
	if !ok {
	    // Handle error.
	}
```

If the result can not be casted to WalletTransactionResult, it will panic.

Reply:
Thank you for the review :+1: